chore: clarify that ASP signs funding PSBT (#8)
ark-network/arkdev.info: docs/learn/concepts.md

- ASP adds the added output in the [Funding transaction](#funding-transaction) shared by Alice.
- ASP spends from the `Alice + ASP` cooperative path.
- ASP adds the output with the script `(Alice + ASP) or (Alice after 24h)`. This output is the [VTXO](#vtxo) of Alice.

> - ASP signs the Funding PSBT and sends it to Alice.

Reviewer: no this is not correct, first is the Redeem transaction to be signed. After this, Alice can sign his original Funding. 

1. Revert this or change it to Redeem PSBT
2. In the next sentence we canbe more clear saying "Is now safe for Alice **to sign** and broadcast his funding.."

```suggestion
- ASP signs the Redeem PSBT, sends it to Alice who counter-signs.
```
